big-o-notation/big-o.js

```diff
@@ -0,0 +1,47 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {}; // 1 * 1 = O(1)
+  const unique = []; // 1 * 1 = O(1)
+  for (
+    let i = 0; // 1 * 1 = O(?)
+    i < words.length; // 2 * n = O(2n) ~= O(n)
+    i++ // 2 * n = O(2n) ~= O(n)
+  ) {
+    const word = words[i]; // 2 * n = O(?)
```

Comment:
These are all `O(n)`, which you got on line 8, so I assume you just didn't update these ones?